src/stan/lang/grammars/semantic_actions_def.cpp

```diff
@@ -1454,13 +1454,24 @@ namespace stan {
     validate_non_void_expression_f;
 
 
-    void add_line_number::operator()(statement& stmt,
+    void add_stmt_line_number::operator()(statement& stmt,
                                      const pos_iterator_t& begin,
                                      const pos_iterator_t& end) const {
       stmt.begin_line_ = get_line(begin);
       stmt.end_line_ = get_line(end);
     }
-    boost::phoenix::function<add_line_number> add_line_number_f;
+    boost::phoenix::function<add_stmt_line_number>
+    add_stmt_line_number_f;
+
+    void add_var_decl_line_number::operator()(var_decl& var_decl,
```

Comment:
used template parameters to define function `add_line_number`, defined for both var_decls and statements.